package.json

```diff
@@ -207,5 +205,8 @@
     "type": "opencollective",
     "url": "https://opencollective.com/verdaccio",
     "logo": "https://opencollective.com/verdaccio/logo.txt"
+  },
+  "dependencies": {
+    "react-router-dom": "5.1.2"
```

Comment:
small reminder, this should be a `devDep`

Reply:
I do not agree. `react-router-dom` Is not a dev tool or a compiler, its code is being used in the compiled, production bundle and thus is a dependency (living with `react` and `react -dom`).

As a rule, anything that is imported by the source file should be a dependency.

Reply:
I agree with Juan...it should be in the `devDep`. I tried and could not find any react-router-dom in the bundle folder...

Reply:
What do you mean ? `react-router-dom` is used during the application run time so it should definitely stay in the dependencies, not dev dependencies.

Dev dependencies are for tooling around the project, the application router is not tooling, it is a library used by the production code :)